.github/workflows/include-kcl-checksums.yml

```diff
@@ -0,0 +1,48 @@
+name: Include KCL Modules Checksum
+
+on:
+  workflow_dispatch:
+    inputs:
+      kpm_reg:
+        description: "Enter the KPM Registry"
+        required: true
+        default: "localhost:5001"
+      kpm_repo:
+        description: "Enter the KPM Repository"
+        required: true
+        default: "test"
+
+jobs:
+  include_modules_checksum:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Install kcl
+        run: wget -q https://kcl-lang.io/script/install-cli.sh -O - | /bin/bash
+
+      - name: Checkout code
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+
+      - name: Set up Go
+        uses: actions/setup-go@v5
+        with:
+          go-version-file: go.mod
+
+      - name: Run local registry for localhost
+        if: ${{ github.event.inputs.kpm_reg == 'localhost:5001' }}
+        run: ./scripts/reg.sh
+
+      - name: Login to registry
+        run: |
+          kcl registry login -u ${{ github.event.inputs.kpm_repo }} -p ${{ secrets.REGISTRY_ACCESS_TOKEN }} ${{ github.event.inputs.kpm_reg }}
```

Comment:
This login command is incorrect, and not all account names are oci repo.

Reply:
Then should I take account name as input or as a secret??

Reply:
Besides the username, can you point out what is incorrect in the command? I tested it locally, and it works fine.Can you also provide a link to the correct command?

Reply:
The user name is not `${{ github.event.inputs.kpm_repo }}`, you can test successfully because in the test case, the user name and `${{ github.event.inputs.kpm_repo }}` happens to be the same, called `test`, but the user name is not necessarily the same as `${{ github.event.inputs.kpm_repo }}`.

Reply:
> Then should I take account name as input or as a secret??

also can you specify this?

Reply:
Do not input this part because this may cause the password to appear in plain text in the GitHub action log; you can try to configure this part using the GitHub action secrets management. 

And, if you have been tested to be able to block input in the github action log in some way, you can also enter your account and password as input.